src/index.js

```diff
@@ -146,13 +146,13 @@ async function main(args) {
   for (const coverageFile of coverageFilePaths) {
     let fileContents
     try {
-      log(`Processing ${coverageFile}...`),
+      info(`Processing ${coverageFile}...`),
         (fileContents = await fileHelpers.readCoverageFile(
           args.dir || projectRoot,
           coverageFile,
         ))
-    } catch (error) {
-      log(`Could not read coverage file (${coverageFile}): ${error}`)
+    } catch (err) {
+      info(`Could not read coverage file (${coverageFile}): ${err}`)
```

Comment:
@thomasrockhu I think this should be an error, but I didn't want to redo the tests. Also, if this is an error, why do we continue? Can we reword this to _potential_ coverage file? If we tell a customer we can't read a coverage file, that should be an error, in my opinion.

Reply:
yeah I agree that it should be `potential`